### PR TITLE
Replace sketches in Approach new task

### PR DESCRIPTION
**Pull request by liatsade**

### CODE REVIEW

File: approach_data_structure_task.md [hunk 1]
```diff
@@ -51,7 +50,7 @@ What is this new topic? Simple - It's an entity, represented as a circle.
 How does `topic` relate to the other entities?
 Articles are written on particular topics, so the `article` and the `topic` have a relationship.
 
-![](images/approach_data_structure_task/4.jpg)
+![](images/approach_data_structure_task/articles-relationship-topic.png)
```
Comment (liatsade) [inline approach_data_structure_task.md:53]: @OritGeron it needs to be arrows from the `articles` to the `author` (article - topic is fine without arrows) Can you draw it again?

wip__replace_sketches_in_approach_new_task_by_liatsade_ _pull_request__46_ _gizra_gizra-way-book

File: approach_data_structure_task.md [hunk 2]
```diff
@@ -66,7 +65,7 @@ Question 1: Can one `article` be written in more than one `topic`? Yes, that is
 Question 2 (The Million Question): Can and should one single `article` refer to a million `topic`?
 This is where reality dictates the answer. While in theory an article could reference a million topics, we know that this won't be the case. A typical article (blog post) will probably have a single or few topics. So, it is safe to say, that based on the limitation which is derived from the fact we are building _real_ sites and not answering academic papers, the `article` refers to the `topic`.
 
-![](images/approach_data_structure_task/5a.jpg)
+![](images/approach_data_structure_task/articles-reference-topic.png)
```
Comment (liatsade) [inline approach_data_structure_task.md:68]: @OritGeron you missed one arrow from `article` to `author`

wip__replace_sketches_in_approach_new_task_by_liatsade_ _pull_request__46_ _gizra_gizra-way-book

**Comment by OritGeron** [comment]
_no_subject__-_orit_gizra_com_-_gizra_ltd__mail

**Comment by OritGeron** [comment]
@liatsade and this is the 2nd one

_no_subject__-_orit_gizra_com_-_gizra_ltd__mail

**Comment by liatsade** [comment]
@amitaibu all sketches have been replaced.

We said we want to represent the properties (`status` and `create date`) of the `membership` in a different way (not in a circle), but I didn't see the old sketch in the post - is it still relevant?

**Comment by amitaibu** [comment]
Thanks.

>  is it still relevant?

Not relevant.

**Pull request merged**